include/osgGA/FreeCameraManipulator

```diff
@@ -0,0 +1,64 @@
+#pragma once

```

Comment:
`#pragma once` is not standard c++ and should be removed. You have already include guards so they serve very little benefit.